packages/devtools/lib/src/trees.dart

```diff
@@ -72,6 +72,18 @@ class TreeNode {
     child.index = children.length - 1;
   }
 
+  bool containsChildWithCondition(bool condition(TreeNode _), {TreeNode root}) {
```

Comment:
add a typedef for Condition

Reply:
probably bad style to use `_` as the variable name here. `node` would be more appropriate. `_` indicates that a variable is ignored.

Reply:
done.